test/runtests.jl

```diff
@@ -97,6 +91,7 @@ end
 
         # Test spatial plotting, using CarioMakie and GraphMakie
         @time @safetestset "Lattice Simulation Plotting" begin include("extensions/lattice_simulation_plotting.jl") end
+        @time @safetestset "Graph visualization" begin include("extensions/graphmakie.jl") end
```

Comment:
Isn't this already in the extensions test section? (Which is more appropriate for it.)